src/internal/DotStarGenericMethod.h

```diff
@@ -29,8 +29,8 @@ License along with NeoPixel.  If not, see
 #include "TwoWireBitBangImple.h"
 
 //TODO remove these once Linux SPI tested and works OK
-#define MOSI  9
-#define SCK   11
+#define MOSI  9//spi0mosi/gpio10/19 ue
```

Comment:
it looks like none of the changes here are needed. In the future, you'll have to remove files like this from the review. since there is no real change

Reply:
Sure will do that